x-pack/platform/plugins/shared/lens/public/editor_frame_service/error_helper.tsx

```diff
@@ -10,6 +10,21 @@ import { renderSearchError } from '@kbn/search-errors';
 import React from 'react';
 import type { UserMessage } from '@kbn/lens-common';
 
+export const ignoredBlockingMessagesByID = new Set<string>(['The expression was aborted.']);
```

Comment:
Can we make this a bit less fragile and add a constant referring to the expression abort message?

https://github.com/dej611/kibana/blob/7e1b519a7ae5fad92ab51d12a1cd4ddda1ed60c8/src/platform/plugins/shared/expressions/common/execution/execution.ts#L91

We don't need to export this `ignoredBlockingMessagesByID`, I believe we are not even going to add more things there